lib/src/impl_ffi/impl_ffi.utils.dart

```diff
@@ -43,30 +40,13 @@ class _EvpPKey {
   /// [use]. Otherwise, the garbage collect may be calling the finalizer while
   /// the key is in use.
   _EvpPKey.wrap(this._pkey) {
```

Comment:
Nit: from a resource perspective, I'd make `wrap` the unnamed constructor. And `new` or `allocate` a factory.